src/Image.php

```diff
@@ -49,37 +49,37 @@ class Image
     /**
      * @var bool whether the PDF was created
      */
-    protected $_isCreated = false;
+    protected $isCreated = false;
```

Comment:
I'm not so happy with removing the `_` from all private/protected vars. Is this part of PSR?